server/src/test/java/io/druid/server/http/DatasourcesResourceTest.java

```diff
@@ -193,4 +181,82 @@ public void testSimpleGetTheDataSource() throws Exception
     Assert.assertNotNull(result.get("segments").get("maxTime").toString(), "2010-01-02T00:00:00.000Z");
     EasyMock.verify(inventoryView, server);
   }
+
+  @Test
+  public void testGetSegmentDataSourceIntervals(){
+    server = new DruidServer("who", "host", 1234, "historical", "tier1", 0);
```

Comment:
this is a general comment that spacing would help a lot with actually reading code